easybuild/easyblocks/m/mvapich2.py

```diff
@@ -70,12 +70,24 @@ def configure_step(self):
 
         # enable specific support options (if desired)
         if self.cfg['withmpe']:
-            add_configopts.append('--enable-mpe')
+            # --enable-mpe is coming from MPICH.
```

Comment:
@boegel  Yes, it makes sense. I can do so.

However, it also means that we have to do the check for the version in the 2 easyblocks (mpich and mvapich2), like we did for the `use_new_libnames` (cf PR https://github.com/hpcugent/easybuild-easyblocks/pull/844). This brings more complexity. Do we want to go for this?

One other solution to unify all this would be to add a extra option `mpich_version` in the MPICH easyblock. Then, every derived easyconfig would have to declare the MPICH version it's based on. For example, in a MVAPICH2 easyconfig, we would have:
```
name = 'MVAPICH2'
version = '2.0.1'
mpich_version = '3.1'

...
```
Then the version check for the MPICH features can be done using `mpich_version` instead of `version`. This way the derived easyblocks do not need to know anything. What do you think?